roles/docker/tasks/main.yml

```diff
@@ -24,5 +24,12 @@
   shell: systemctl daemon-reload && systemctl stop docker && systemctl start docker-tcp.socket && systemctl start docker
   when: docker_env_conf_result.changed or docker_tcp_conf_result.changed or docker_svc_conf_result.changed
 
+#NOTE: Without this step, pip crashes after installing docker-compose
+#      Move to appropriate role later
+- name: upgrade pip
```

Comment:
the `base` role already installs pip. This task should not be needed if `base` is run before running `docker` role.